packages/__docs__/src/Document/index.tsx

```diff
@@ -71,7 +72,10 @@ class Document extends Component<
     this.props.makeStyles?.()
   }
 
-  handleDetailsTabChange = (_event: Event, { index }: any) => {
+  handleDetailsTabChange = (
+    _event: React.MouseEvent<ViewOwnProps> | React.KeyboardEvent<ViewOwnProps>,
+    { index }: any
+  ) => {
```

Comment:
Since this has to be the exact same as the `onRequestTabChange` prop of Tabs, you can use it's type:

```js
handleDetailsTabChange: TabsProps['onRequestTabChange'] = (
    _event,
    { index }
  ) => {
    this.setState({
      selectedDetailsTabIndex: index
    })
  }
```